canonical/ubuntu.com
# WD-12853 - rebrand /gcp/fips

## Done

Rebrand  /gcp/fips. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14249.demos.haus/gcp/fips)
- [figma](https://www.figma.com/design/vz3yAG1nYI7r5gc3SpTUOw/ubuntu.com%2Fgcp%2Ffips?node-id=1-3103&node-type=FRAME&t=p9GnMvla4VmAGUIl-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12853](https://warthogs.atlassian.net/browse/WD-12853)

[WD-12853]: https://warthogs.atlassian.net/browse/WD-12853?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
